src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -318,10 +280,12 @@ function PaymentMethodList({
                     icon={item.icon}
                     disabled={item.disabled}
                     displayInDefaultIconColor
+                    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
                     iconHeight={item.iconHeight || item.iconSize}
```

Comment:
@filip-solecki What do you think? Should we reflect the old code directly, or assume that the original authors only meant to check for `undefined` with `||`?

@tienifr Did you search for the usages? Do we have a convention to pass `0` as height/length meaning "not set"?

Reply:
Imo we should check only for undefined, 0 should be passed intentionally, but don't know what is the convention you mentioned

Reply:
I mean that there's _sometimes_ a convention that 0 could mean "not set", like in the case of some IDs. Then, silencing the linter would make sense, and we could use `||` on purpose. If it's not the case, I think we should migrate to `??` and _not_ silence the linter in this specific case.

Reply:
@tienifr Let's adjust as @cubuspl42 suggested, and let's continue the work to ship it!

Reply:
Thanks for bumping, and I agree that we should wrap things up!

@tienifr

Reply:
Great!